README.md

```diff
@@ -71,6 +71,23 @@ brew install cmdg
 ```
 
 ## Configuring
+You need to configure `cmdg` in order to provide it with authentication
+so it can talk to GMail on your behalf. To do this you'll need to generate
+a ClientID and ClientSecret. You can do this with the following steps:
+
+  1. Go to the [Google Developers Console](https://console.developers.google.com/apis).
+  1. Select an existing project or create a new project
+  1. Navigate to the "Credentials" tab.
```

Comment:
I'm not sure this accurately describes the steps. You did it more recently though.

Why navigate to the credentials tab (you mean page?) and then immediately go to the OAuth consent screen, which is a different page in the nav on the left from Credentials?

Reply:
You're right, that wasn't clear. I've simplified the steps in the latest PR